docs/user/set-up/quick-starts.md

```diff
@@ -0,0 +1,26 @@
+---
+aliases:
+- /docs/agent/latest/set-up/quick-starts/
+title: Grafana Agent quick starts
+weight: 150
+---
+## Grafana Agent quick starts
+
+The following quick starts help you get up and running with Grafana Agent. You’ll learn how to send your metrics, logs, and traces to the Grafana Stack or Grafana Cloud.
+
+### Grafana Stack quick starts
+
+ - Send metrics to Mimir using Grafana Agent
+https://grafana.com/docs/mimir/latest/operators-guide/getting-started/
+
+ - Send traces to Tempo using Grafana Agent https://grafana.com/docs/tempo/latest/getting-started/#2-pipeline-grafana-agent
+
+ - Send logs to Loki using Grafana Agent https://grafana.com/docs/loki/latest/clients/promtail/ (https://grafana.com/docs/grafana-cloud/logs/collect-logs-with-agent/?pg=hp&plcmt=lt-box-traces
```

Comment:
There's a stray parenthesis here. Also, do we want to turn these into proper hyperlinks instead of URLs?

Reply:
Yes--will do